test/multiverse/suites/logstasher/Envfile

```diff
@@ -0,0 +1,18 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+instrumentation_methods :chain, :prepend
+
+logstasher_versions = [
+  [nil, 2.7]
+]
+
+def gem_list(logstasher_versions = nil)
+  <<~RB
+    gem 'logstasher'#{logstasher_versions}
+    gem 'activesupport', '~> 7.0.0'
```

Comment:
From your last explanation of why we need to bring in ActiveSupport, I believe you really just want to enforce a minimum version. We should be safe to test with v7.1 that's out now and v8.0 in future. How about adding a comment here that reminds of why we need to bring in ActiveSupport, and how about simply using `>=`?

```suggestion
    gem 'activesupport', '>= 7'
```

Reply:
When running a playground locally, there doesn't seem to be an issue. But multiverse tests have an issue with versions above 7.1 that I couldnt figure out/ not sure if it was worth the effort (its a `NoMethodError` coming from activesupport 7.1)

Reply:
Ah, I got it flipped then. You would need `< 7.1`, not `>= 7`. Ok, if it's a ceiling and not a minimum, then perhaps the code can stay as-is but just get the comment reminding us of the problem. Logstasher itself should have its gemspec updated to denote the <7.1 constraint.